tests/cron_job_tests/test_offline_app_setup.py

```diff
@@ -79,3 +80,18 @@ def test_genomics_config_disables_jobs(self, pipeline_mock):
     def test_biobank_missing_samples_check_route(self, mock_checker):
         self.send_cron_request('BiobankMissingSamplesCheck')
         mock_checker.assert_called()
+
+    @mock.patch('rdr_service.offline.main.dispatch_check_consent_errors_task')
+    def test_consent_error_report_route(self, mock_checker):
+        from unittest.mock import call
```

Comment:
Does this work if the import is at the top of the file? (mostly just curious why the import is in the method)

Reply:
I'd seen other instances where narrowly used imports were local to the specific place where they were needed.  I can move it if you want.

Reply:
I can't speak for all the code, but I think when I've done an import in the method it was only when needed (to avoid a circular dependency when trying to load files). Otherwise it seems to be a best practice to have the imports at the top (at the very least for having a standard place for knowing what dependencies a file has).

Alternatively, you could use `mock.call` to take advantage of something already present.

Reply:
Resolved in latest commit